### PR TITLE
Update Customer Satisfaction to ZenML 0.20.2

### PR DESCRIPTION
**Pull request by safoinme**
## Describe changes
Update the custom satisfaction ZenFile to work with ZenML 0.20.2!

## Pre-requisites
Please ensure you have done the following:
- [X] If my change requires a change to docs, I have updated the documentation accordingly.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Other (add details above)

### CODE REVIEW

File: customer-satisfaction/README.md [hunk 1]
```diff
@@ -20,6 +20,16 @@ cd zenfiles/customer-satisfaction
 pip install -r requirements.txt
 ```
 
+Starting from ZenML 0.20 ZenML comes bundled with React-based dashboard. This has dashboard will allow you
```
Comment (htahir1) [inline customer-satisfaction/README.md:23]:
```suggestion
Starting from ZenML 0.20.0 ZenML comes bundled with React-based dashboard. This has dashboard will allow you 
```

File: customer-satisfaction/materializer/custom_materializer.py [hunk 2]
```diff
@@ -58,7 +58,7 @@ def handle_return(
         self,
         obj: Union[
             str,
-            np.ndarray,
+            numpy.ndarray,
```
Comment (htahir1) [inline customer-satisfaction/materializer/custom_materializer.py:61]: Why this change?

File: customer-satisfaction/run_pipeline.py [hunk 3]
```diff
@@ -7,20 +7,17 @@
 from zenml.integrations.mlflow.mlflow_utils import get_tracking_uri
```
Comment (htahir1) [inline customer-satisfaction/run_pipeline.py:7]: Would it make sense to run from a YAML file instead using the new method described here: https://docs.zenml.io/advanced-guide/pipelines/settings#method-3-configuring-with-yaml

**Review by htahir1 — CHANGES_REQUESTED** [review]
Nice one! Left a few comments

File: customer-satisfaction/materializer/custom_materializer.py (hunk 2 above)
Comment (safoinme) [reply customer-satisfaction/materializer/custom_materializer.py:61]: I was facing an error with the materializer which didn't make sense! I was just modifying a lot of stuff and didn't revert this back! will do it right now

**Review by htahir1 — COMMENTED** [review]
Looks good now but might make sense to add more details of running with the config file in the README?

**Review by strickvl — CHANGES_REQUESTED** [review]
Some small changes in this review, but the biggest thing currently is that the deployment pipeline doesn't seem to work for me. Let's try again together tomorrow at some point?

**Review by strickvl — CHANGES_REQUESTED** [review]
One other change I'd make is to increase the pinned zenml version to 0.20.5 (i.e. the current latest version) since we know it works.

Otherwise I've tested this on my end and it works, so nice job on making that work again!
